### DISCUSSION TITLE
What's the difference between `parserOptions/ecmaVersion:13` vs. `env/es2021:true`?

### DISCUSSION THREAD

**Discussion by pubmikeb** [eslint/eslint#15166] [Q&A] [answered]
In `eslintrc.json` there are two similar properties:

```json
"env": {
	"es2021": true
},
"parserOptions": {
	"ecmaVersion": 13
}
```

Isn't a redundancy? What's the reason to keep both properties?
What happens when `ecmaVersion` contradicts with `es20xx` property?
E.g. `"ecmaVersion": 13` and `"es2007": true`.

**Comment by nzakas** [comment, marked as answer]
parserOptions indicates the syntax you want to allow. The es2022 environment indicates that you want to add the ES2022 global variables.

**Reply by melroy89** [reply]
es2021* you mean? What will do es202**2**?

**Reply by nzakas** [reply]
"es2022": true :smile:
